nncf/quantization/algorithms/accuracy_control/evaluator.py

```diff
@@ -101,7 +118,16 @@ def prepare_model_for_inference(self, model: TModel) -> TPModel:
         :param model: A model that should be prepared.
         :return: Prepared model for inference.
         """
-        return self._algo_backend.prepare_for_inference(model)
+        backend = get_backend(model)
+
+        if backend == BackendType.OPENVINO:
```

Comment:
Why the `if` condition and not `OVAccuracyControlAlgoBackend.prepare_for_inference(model)`?